src/lib/y2partitioner/widgets/blk_devices_table.rb

```diff
@@ -260,20 +262,26 @@ def type_icon(device)
       end
 
       DEVICE_LABELS = {
-        bcache:        N_("Bcache"),
-        disk:          N_("Disk"),
-        dasd:          N_("Disk"),
-        multipath:     N_("Multipath"),
-        nfs:           N_("NFS"),
-        bios_raid:     N_("BIOS RAID"),
-        software_raid: N_("RAID"),
-        lvm_pv:        N_("PV"),
-        lvm_vg:        N_("LVM"),
-        lvm_lv:        N_("LV"),
-        stray:         N_("Xen"),
-        thin_pool:     N_("Thin Pool"),
-        thin:          N_("Thin LV"),
-        partition:     N_("Partition")
+        bcache:         N_("Bcache"),
+        disk:           N_("Disk"),
+        dasd:           N_("Disk"),
+        multipath:      N_("Multipath"),
+        nfs:            N_("NFS"),
+        bios_raid:      N_("BIOS RAID"),
+        software_raid:  N_("RAID"),
+        lvm_pv:         N_("PV"),
+        lvm_vg:         N_("LVM"),
+        lvm_thin:       N_("Thin LV"),
+        lvm_thin_pool:  N_("Thin Pool"),
+        lvm_raid:       N_("RAID LV"),
+        lvm_cache:      N_("Cache LV"),
+        lvm_cache_pool: N_("Cache Pool"),
+        lvm_writecache: N_("Writecache LV"),
+        lvm_snapshot:   N_("Snapshot LV"),
+        lvm_mirror:     N_("Mirror LV"),
+        lvm_lv:         N_("LV"),
+        stray:          N_("Xen"),
+        partition:      N_("Partition")
```

Comment:
The plan still is to extract this to a separate class, isn't it?

Reply:
Yes, it is. I'm on it.